templates/server-service.yaml

```diff
@@ -15,6 +15,9 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.server.frontend.service.annotations "context" $) | nindent 4 }}
   {{- end }}
 spec:
+  {{- if .Values.server.frontend.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.server.frontend.service.loadBalancerIP }}
```

Comment:
this only works if `.Values.server.frontend.service.type` is `LoadBalancer` - can we check both things for safety?